services/navigation/builtin/builtin_test.go

```diff
@@ -232,7 +234,7 @@ func TestNew(t *testing.T) {
 		test.That(t, svcStruct.storeType, test.ShouldEqual, string(navigation.StoreTypeMemory))
```

Comment:
[note] add test cases for Validate for new obstacle_detector (vision and/or camera not present). See comment above for details.